downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -62,6 +81,30 @@ public function main() {
 
 ### Improvements
 
+#### Improved resolving logic of module level constructs.
+
+With this new feature,
+
+- Made the type node in constant declarations optional. Both of the following cases are valid.
+
+    ```ballerina
+    const map<string> PERSONA = {firstName: "Emily", lastName: "Johnson"};
+    const PERSONB = {firstName: "David", lastName: "Thompson"};
+    ```
+  
+- Allowed union types as valid type nodes for constant declarations.
+
+    ```ballerina
+    const float|decimal TOTAL = 1 + 2.0;
+    ```
+
+- Revamped the logic for resolving module type definitions to properly handle cyclic type definitions.
```

Comment:
I would split this into 3 separate improvements => 3 headings.

Reply:
Maybe first two can be included under same heading like `Improve constant resolving logic`. WDYT?
@MaryamZi

Reply:
It's two different things that add value to the user, right? So it's OK to have two different points IMO.

Reply:
Okay. I will update that.